crates/typst-cli/src/args.rs

```diff
@@ -132,8 +133,53 @@ pub struct SharedArgs {
         value_parser = clap::value_parser!(DiagnosticFormat)
     )]
     pub diagnostic_format: DiagnosticFormat,
+
+    /// Add an input value visible through sys.inputs
+    /// whose type is always treated as a string
```

Comment:
Two things:
- I'm not sure how longer comments render in clap. If it looks okay, it's fine, but this is what has stopped me so far from adding too much docs here.
- The sentence doesn't read well with the "whose" and "which". If went want to say all of this, it should be multiple sentences. 
- While I see the benefit of saying that everything is a string, I don't think we need to mention the quotes thing, that's just how shell works after all 

Overall, I feel like this could just be:
```
/// Add an input string visible through `sys.inputs`
```

Reply:
I agree that it is bloat in the CLI, but I think it should be noted in the documentation.
    /// Add an input value visible through sys.inputs
    /// whose type is always treated as a string
    /// which you may enclose with single or double quote